ajax/dropdownDefaultValue.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * -------------------------------------------------------------------------
+ * Fields plugin for GLPI
+ * -------------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of Fields.
+ *
+ * Fields is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Fields is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Fields. If not, see <http://www.gnu.org/licenses/>.
+ * -------------------------------------------------------------------------
+ * @copyright Copyright (C) 2013-2022 by Fields plugin team.
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ * @license   GPLv2 https://www.gnu.org/licenses/gpl-2.0.html
+ * @link      https://github.com/pluginsGLPI/fields
+ * -------------------------------------------------------------------------
+ */
+
+use Glpi\Http\Response;
+
+include('../../../inc/includes.php');
+header("Content-Type: text/html; charset=UTF-8");
+Html::header_nocache();
+Session::checkLoginUser();
+// Read parameters
+$context  = $_POST['context'] ?? '';
+$itemtype = str_replace("dropdown-", "", $_POST["itemtype"]) ?? '';
+/** @global array $CFG_GLPI */
+
+// Check for required params
+if (empty($itemtype)) {
+    Response::sendError(400, "Bad request: itemtype cannot be empty", Response::CONTENT_TYPE_TEXT_HTML);
+    die;
+}
+
+$table = getTableForItemType($itemtype);
+$rand = $_POST["rand"] ?? mt_rand();
+// Message for post-only
+if (!isset($_POST["admin"]) || ($_POST["admin"] == 0)) {
+    echo "<span class='text-muted'>" .
+    __('Enter the first letters (user, item name, serial or asset number)')
+    . "</span>";
+}
```

Comment:
Search field may be not display if elements count is lower than `$CFG_GLPI['ajax_limit_count']`. Also, this dropdown acts the same as other GLPI dropdowns, so adding a hint for this here does not seems really usefull, as fields would probably be configured by admins that are aware of how it should work.